### ISSUE TITLE
[ERROR] Could not resolve "footer" while running remix app

### ISSUE THREAD

**Issue by aymanapatel** [lerna/getting-started-example#20]
When checkout out the repo and doing install in each of packages, when I run `npx lerna run dev --scope=remixapp`, getting this error:

```
> remixapp@ dev D:\lerna-projects\lerna-work-please\getting-started-example\packages\remixapp
> remix dev
Watching Remix app in development mode...
X [ERROR] Could not resolve "footer"
    app/routes/index.tsx:2:21:
      2 │ import {Footer} from 'footer';
        ╵                      ~~~~~~~~
  You can mark the path "footer" as external to exclude it from the bundle, which will remove this error.
Build failed with 1 error:
app/routes/index.tsx:2:21: ERROR: Could not resolve "footer"
```

Node version: `v14.17.0`
Platform: Windows

**Comment by Torres7707** [comment]
the same problem...

**Comment by hajjarjoseph** [comment]
Same problem when trying to follow the getting-started guide.

**Comment by zhangbg** [comment]
I have the same problem by using the lerna@6.6.2（node@14.19.0）

**Comment by zhangbg** [comment]
@aymanapatel You should build the footer and header packages （npx lerna run build --scope=footer） before you run the `npx lerna run dev --scope=remixapp`。

**Comment by MidoAhmed** [comment]
Hello [aymanapatel](https://github.com/aymanapatel) i had the same issue typescipt was not able to resolve definition files what woked for me is to adjust tsconfig.json like this:
- option 1:

  `"baseurl": "../"`

- option 2:
`
"baseurl": "."
"paths": {
      "footer": ["../footer"],
      "header": ["../header"]
    }
`

**Comment by fahslaj** [comment]
Closing this issue as it appears to have been fixed in #18. Feel free to follow up if anyone is still experiencing this issue.
